Allphi/Presentation/Views/ParkingApp.xaml.cs

```diff
@@ -44,12 +44,14 @@ private void Btn_NL_Click(object sender, RoutedEventArgs e)
 
         private void Btn_Submit_Employee_Click(object sender, RoutedEventArgs e)
         {
+            _licensePlate = txtBox_LicensePlate.Text;
+            _dc.SubmitEmployeeParking(_licensePlate);
```

Comment:
Submit in the name is a bit of a weird choice, doesn't tell me what it'll do either.